app/locales/en/translations.js

```diff
@@ -1,22 +1,23 @@
-const preprints = `{{preprintWords.Preprints}}`;
-const brand = `OSF Preprints`;
-
 export default {
     global: {
         cancel: `Cancel`,
         abstract: `Abstract`,
         doi: `DOI`,
         tags: `Tags`,
-        preprints,
-        brand,
-        brand_name: 'OSF',
-        provider_brand: `{{name}} {{preprintWords.Preprints}}`,
         title: `Title`,
         authors: `Authors`,
         license: 'License',
         pre_moderation: `pre-moderation`,
         post_moderation: `post-moderation`,
     },
+    documentType: {
```

Comment:
Seems like this should be in `ember-osf`, replacing its `preprintWords` translations